prow/repoowners/repoowners.go

```diff
@@ -48,11 +50,13 @@ type dirOptions struct {
 }
 
 // Config holds roles+usernames and labels for a directory considered as a unit of independent code
+// along with the path to the OWNERS file that was parsed to retrieve those labels, usernames and roles.
 type Config struct {
 	Approvers         []string `json:"approvers,omitempty"`
 	Reviewers         []string `json:"reviewers,omitempty"`
 	RequiredReviewers []string `json:"required_reviewers,omitempty"`
 	Labels            []string `json:"labels,omitempty"`
+	SourcePath        string   `json:"-"`
```

Comment:
Does this need to be exported? Thus far it doesn't look like it is being used outside the package.

Reply:
Agreed. Now sourcePath is no longer exported.